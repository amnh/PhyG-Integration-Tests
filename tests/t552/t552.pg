--input t5.tre, swap(atrandom), inputting fasta and binary files
set(seed:1634561640)
read(fasta:"chel.fas", "chel_binary.fas")
read(newick:"../t5/t5_nocollapse.tre")
swap(atrandom)
report("t552.csv", data, overwrite)
report("t552.tre", newick, overwrite)