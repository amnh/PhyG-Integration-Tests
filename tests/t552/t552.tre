(Alentus:1.0,(Aportus:6.0,(Thermobius:16.0,(Rhiphicephalus:24.0,(Vonones:13.0,((Limulus:9.0,(Centruoides:10.0,(Hadrurus:10.0,Pauroctonus:1.0)HTU23:10.0)HTU22:7.0)HTU20:1.0,((Chanbria:12.0,(Mastigoproctus:6.0,(Gea:14.0,(Americhernus:28.0,Artemia:14.0)HTU25:2.0)HTU26:9.0)HTU30:2.0)HTU29:3.0,(Amblypygid:10.0,(Hypochilus:2.0,Thelichoris:3.0)HTU32:7.0)HTU31:3.0)HTU28:2.0)HTU27:3.0)HTU21:5.0)HTU24:1.0)HTU19:13.0)HTU18:0.0)HTU17[511.0];
