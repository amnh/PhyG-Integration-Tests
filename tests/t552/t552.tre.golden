(Alentus:28.0,(Aportus:28.0,(Vonones:8.0,(Amblypygid:8.0,((Mastigoproctus:13.0,((Gea:25.0,(Hypochilus:1.0,Thelichoris:4.0)HTU21:4.0)HTU22:3.0,((Chanbria:21.0,Limulus:9.0)HTU20:4.0,(Centruoides:9.0,(Hadrurus:8.0,Pauroctonus:3.0)HTU29:12.0)HTU28:5.0)HTU27:6.0)HTU19:1.0)HTU26:4.0,(Rhiphicephalus:21.0,(Thermobius:13.0,(Americhernus:38.0,Artemia:14.0)HTU31:7.0)HTU32:6.0)HTU30:5.0)HTU25:6.0)HTU23:2.0)HTU24:12.0)HTU18:30.0)HTU17[505.0];
